packages/mdc-fab/_mixins.scss

```diff
@@ -333,6 +333,8 @@ $mdc-fab-icon-enter-duration_: 180ms;
     width: auto;
     max-width: 100%;
     height: $mdc-fab-extended-height;
+    /* @alternate */
```

Comment:
Can you add a code comment about why we do this?

Reply:
Is is mostly an issue with the CSS compilation because there are multiple identical, non-alternate declarations in the same ruleset. line-height is being set twice within `.mdc-fab--extended`, once in `mdc-typography(button, $query: $query)` and then again here. Similar code is done within button/mixins.scss.

Added comment!

Reply:
Sorry, I meant comment for `line-height: inherit` :)

Reply:
No worries, updated the comment 👍